Formula/bazel.rb

```diff
@@ -15,6 +16,9 @@ class Bazel < Formula
 
   def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
+    # Bazel has it owns sandbox, ensure it works.
+    ENV.delete("SDKROOT")
+    ENV["CC"] = `xcrun -f #{ENV["CC"]}`.strip
```

Comment:
Ok stripping SDKROOT was incorrect, replacing it by HOMEBREW_SDKROOT is correct.
(btw, the environment SDKROOT is simply incorrect. I don't know who set it, I don't have it in my environment so it's coming from some part of Homebrew).

There is no way to tweak the action environment in Bazel (that would be against its philosophy). 

IMHO the real correct way would be for the wrapper in the superenv to actually includes those information and make themselves un-distinguishable from a normal compiler (so they don't need those HOMEBREW_* environment variables).

Note that the previous version of bazel was directly coding the path to /usr/bin/gcc in its C++ configuration file (so it's actually a progress toward using the superenv here). We could use an option to use those old configuration file instead (that would be bypassing even more the superenv).